contentctl/objects/risk_event.py

```diff
@@ -1,17 +1,55 @@
 import re
 from typing import Union, Optional
+from functools import cached_property
 
-from pydantic import BaseModel, Field, PrivateAttr, field_validator
+from pydantic import BaseModel, Field, PrivateAttr, field_validator, computed_field
 
 from contentctl.objects.errors import ValidationFailed
 from contentctl.objects.detection import Detection
 from contentctl.objects.observable import Observable
 
+# TODO (cmcginley): validation of threat objects
+# TODO (cmcginley): 'Email' -> other is where it goes in practice, but I feel like it SHOULD go to
+#   user?
+# TODO (cmcginley): Should we map some of these others more specifically? E.g. 'Process' to
+#   'host_artifacts' and 'File Hash' to 'hash_values'
+# TODO (cmcginley): centralize this mapping w/ usage of SES_OBSERVABLE_TYPE_MAPPING (see
+#   observable.py) and the ad hoc mapping made in detection_abstract.py (see the risk property func)
+
 # TODO (PEX-433): use SES_OBSERVABLE_TYPE_MAPPING
 TYPE_MAP: dict[str, list[str]] = {
```

Comment:
Should we move some of the observable types into more appropriate mappings? e.g. `File Hash` to `hash_values`

Reply:
`File Hash` under `hash_values` makes sense to me.  There are a few that I am a little less certain about. 
is `MAC Address` best as a system (IP Address is a system artifact, but MAC address is even more closely tied to a system than that), network_artifact, host_artifact, or other?
Lots of other things here, like why shouldn't registry stuff be a host_artifact? I will reach out to Lou and get his thoughts and link to this comment :)

Reply:
Cool! I'm gonna make this a follow up issue though for now, so it doesn't block this PR

Reply:
#259